src/main/java/core/basesyntax/service/filesoperation/FileReaderImpl.java

```diff
@@ -0,0 +1,21 @@
+package core.basesyntax.service.filesoperation;
+
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.List;
+
+public class FileReaderImpl implements FileReader {
+    private static final String ERROR_READ_MESSAGE = "Can't find file by path: ";
+    private List<String> frits;
+
+    @Override
+    public List<String> read(String filePath) {
+        try {
+            frits = Files.readAllLines(Path.of(filePath));
+        } catch (IOException e) {
+            throw new RuntimeException(ERROR_READ_MESSAGE + filePath);
```

Comment:
```suggestion
    @Override
    public List<String> read(String filePath) {
        List<String> fruits;
        try {
            fruits = Files.readAllLines(Path.of(filePath));
        } catch (IOException e) {
            throw new RuntimeException("Can't find file by path: "+ filePath);
```
fruits variable should be local